vertx-pojo-mapper-common-test/src/main/java/de/braintags/io/vertx/pojomapper/dataaccess/query/TestFieldConditionCache.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * #%L
+ * vertx-pojo-mapper-json
+ * %%
+ * Copyright (C) 2017 Braintags GmbH
+ * %%
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * #L%
+ */
+package de.braintags.io.vertx.pojomapper.dataaccess.query;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import de.braintags.io.vertx.pojomapper.dataaccess.query.impl.FieldCondition;
+import de.braintags.io.vertx.pojomapper.dataaccess.query.impl.IQueryExpression;
+import de.braintags.io.vertx.pojomapper.dataaccess.query.impl.VariableFieldCondition;
+import de.braintags.io.vertx.pojomapper.testdatastore.DatastoreBaseTest;
+import de.braintags.io.vertx.pojomapper.testdatastore.mapper.MiniMapper;
+import io.vertx.codegen.annotations.Nullable;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+
+public class TestFieldConditionCache extends DatastoreBaseTest {
```

Comment:
Tests to be executes must be contained inside a TestSuite, here probably in AllTestCommon